core-primitives/stf-executor/src/executor.rs

```diff
@@ -188,7 +188,6 @@ where
 		account: AccountId,
 		amount: Amount,
 		shard: &ShardIdentifier,
-		calls: &mut Vec<OpaqueCall>,
```

Comment:
Removed this input parameter, as there will never(!) be a new OpaqueCall introduced during Stf::execute of a shielding extrsinic. (New OpaquelCall is only issued when calling "unshield")

Reply:
Thanks for clarifying this ☺️

Reply:
Haha, good catch. 🛩️